website_docs/manual.md

```diff
@@ -194,6 +194,33 @@ Events can also have attributes of their own -
 span.AddEvent("Cancelled wait due to external signal", trace.WithAttributes(attribute.Int("pid", 4328), attribute.String("signal", "SIGHUP")))
 ```
 
+### Set span status
+
+A status can be set on a span, typically used to specify that there was an error in the operation a span is tracking - StatusCode.ERROR.
+
+```go
+result, err := operationThatCouldFail()
+if err != nil {
+	span.SetStatus(codes.Error, "operationThatCouldFail failed")
```

Comment:
How about adding the `import` for `codes` here?
I'm always a bit confused when I see a code sample which relies on an external package and doesn't tell where to find it.